website/scripts/pages/forum.js

```
@@ -0,0 +1,398 @@
+
+function showError(message) {
+    // Simple alert for now, could be improved with toast notifications
+    alert('Error: ' + message);
+}
+
+function showSuccess(message) {
+    alert('Success: ' + message);
+}
+
+// Close modal functions
+function closeCreatePostModal() {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'closeCreatePostModal'. Consider to delete it or rename it.

'closeCreatePostModal' is defined here:


(lint/suspicious/noRedeclare)